gqcp/tests/ONVBasis/OperatorString_test.cpp

```diff
@@ -89,6 +125,65 @@ BOOST_AUTO_TEST_CASE(test_unresolved_isZero) {
     BOOST_CHECK_EQUAL(operator_string_2.isZero(), true);
 }
 
+
+BOOST_AUTO_TEST_CASE(test_unresolved_sorting) {
```

Comment:
Maybe add a bit more documentation to this test, just to make it clear for everyone who might work on it.